erpnext/setup/setup_wizard/data/country_wise_tax.json

```diff
@@ -1163,481 +1163,6 @@
 		}
 	},
 
-	"India": {
```

Comment:
How is this created in the new app? Some users might not have the India compliance app before setup, if they choose India in the setup wizard what COA will the first company have?

Reply:
We added a event using hooks in the Company doctype's `on_update` method. It checks if `frappe.flags.country_change` is set (from ERPNext) and installs these defaults based on that.

Reply:
In case of installing the app after creating the company, user can click a button in their Company doctype to trigger this manually.

Reply:
However, I can also build a way to detect if the defaults are set for existing companies when India Compliance is installed and then if they are not set, I can create them on install.

Reply:
solved in https://github.com/resilient-tech/india-compliance/pull/152